include/merlin/core_kernels.cuh

```diff
@@ -1064,9 +1062,8 @@ __forceinline__ __device__ void update_score(Bucket<K, V, S>* __restrict bucket,
                                              const S* __restrict scores,
                                              const int key_idx) {
   if (scores == nullptr) {
-    S cur_score =
-        bucket->cur_score.fetch_add(1, cuda::std::memory_order_relaxed) + 1;
-    bucket->scores(key_pos)->store(cur_score, cuda::std::memory_order_relaxed);
+    bucket->scores(key_pos)->store(device_nano<S>(),
```

Comment:
`kLru` is faster than `kCustomized` in benchmark. Is it because that the src loaded is changed from L2-ram to register?

Reply:
Yes, I think so. Plus, the `fetch_add` on the `score` for one key should be not enough to fill the cache line, so the result is also benefits from canceling it.